src/test-utils/events.ts

```diff
@@ -18,3 +18,7 @@ export function createEventLogger() {
 
   return { events, logEvent };
 }
+
+export function getEventsName(events: EventEntry[]) {
```

Comment:
Is it worth it? Maybe we could expose `eventNames` next to `events` in the `createEventLogger` return. Wdyt?

Reply:
I find it worth it yes, after writing the same map several times I felt we needed an additional helper to do this. I like the idea of returning `eventNames` directly from `createEventLogger ` though, I'll make the change

Reply:
Actually I'm not sure it's a good idea because I can't return directly the eventNames because it wouldn't be updated whenever we push a new event, unless we'd have two arrays but that would be having two sources of truth so it does not look like a great idea. Another solution would be to make it so that `createEventLogger` returns a function getEventsName. It would probably be a bit better because we don't need to pass events as a parameter to the function, wdyt?

Reply:
Current version is ok.